# for visualization
'path_red' : 1.0
'path_green' : 0.0
'path_blue' : 1.0
'path_alpha' : 0.75
'min_collision' : 0.4
'octomap_test_file' : "/home/fahri/data/fr_078_tidyup.bt"